string name
string uuid
time creationTimestamp
string data